lib/emoji.rb

```diff
@@ -63,19 +63,24 @@ def edit_emoji(emoji)
 
   # Public: Find an emoji by its aliased name. Return nil if missing.
   def find_by_alias(name)
-    names_index[name]
+    @mutex.synchronize do
```

Comment:
@myxaluch Yes, please open an alternative PR with the approach of preloading all emoji at load. I think that's a much safer and simpler approach, and would make multithreading faster since no mutex will be needed.